### ISSUE TITLE
Invite link hard to find

### ISSUE THREAD

**Issue by benkinooby** [telegramdesktop/tdesktop#6208]
Telegram Desktop v. 1.7.10 hides the invite link in a hard to find place.

In version 1.7.10, you can generate (or find your current) invite link in:
**manage group -> group type**

Please move the "invite link" functionality away from the obscure "group type" menu to either "manage group" or "add member", where it was before and was intuitive to find.

**Comment by john-preston** [comment]
@benkinooby It was never in the "add member" on Telegram Desktop :( But it should be indeed.

**Comment by benkinooby** [comment]
> @benkinooby It was never in the "add member" on Telegram Desktop :( But it should be indeed.

Yeah, I was not sure where it was exactly. Afaik, the mobile version has it at "add member" so it would make sense to put it there in the desktop version as well for consistency.

**Comment by benkinooby** [comment]
This is still an issue.

One would expect an invite link in "Add members" or "manage group". Don't "hide" that link in "manage group" -> "group type".
Making that link easy to find and share is important.

**Comment by benkinooby** [comment]
@23rd I was looking for the commit completing this issue but was not able to find/identify it. Can you link the commit(s), that complete this isssue, here? Thank you in advance.

**Comment by 23rd** [comment]
@benkinooby 
[02ad5f2772109b9535fd5e0b20d6214aea15849b](https://github.com/telegramdesktop/tdesktop/commit/02ad5f2772109b9535fd5e0b20d6214aea15849b)

**Comment by benkinooby** [comment]
Thank you - invite links (on Telegram Desktop) can be found at:
* directly at "manage group" 
* at "add members"

Thank you!
